code/modules/research/designs/anomaly.dm

```diff
@@ -0,0 +1,29 @@
+/datum/design/scienceglasses
+	name = "Science Goggles"
+	desc = "You expect those glasses to protect you from science-related hazards. Maybe you shouldn't."
+	id = "scienceglasses"
+	req_tech = list(Tc_MATERIALS = 1, Tc_ANOMALY = 2)
+	build_type = PROTOLATHE
+	materials = list(MAT_IRON = 700, MAT_GLASS = 2000)
+	category = "Anomaly"
+	build_path = /obj/item/clothing/glasses/science
+
+/datum/design/depth_scanner
+	name = "Depth Analysis Scanner"
+	desc = "Used to check spatial depth and density of rock outcroppings."
+	id = "depth_scanner"
+	req_tech = list(Tc_MATERIALS = 1, Tc_ANOMALY = 2)
+	build_type = PROTOLATHE
+	materials = list(MAT_IRON = 500, MAT_GLASS = 1000, MAT_GOLD = 200, MAT_SILVER = 200)
+	category = "Anomaly"
+	build_path = /obj/item/device/depth_scanner
+
+/datum/design/ano_scanner
```

Comment:
> Implying that jumping into space hoping to find some fun, or loitering on the asteroid only to come back on the station and wonder what happened isn't part of SS13's core concepts. 

No, it really isn't. I understand that SS13 means a lot of different things to a lot of different people but of all the mechanics SS13 has, space exploration is one of the worst developed ones, even if the rest aren't that great either.

Not to mention that space exploration and such directly contradict player interaction, and you're gonna need a really good argument to say that player interaction isn't a core part of SS13. Not to mention that if you want space exploration, there are much better games for it.

> loitering on the asteroid only to come back on the station and wonder what happened

I don't consider this part of the "core concepts" because, let's be honest, no matter what happened outside of adminbus it's extremely easy to figure out what happened in 99% of cases.

> There's enough room on the server for both the kind who wants to interact and the one that doesn't.

A lot of people think because they don't actively brainstorm about it. Even if you consider mining to be fun, you need some serious autism to hate interaction that much that you don't like potential alternatives or reworks like a mineral magnet (goon). I can assure you that anything mining does, can be done better with an on-station mechanic.